src/map/pc.cpp

```diff
@@ -10785,6 +10804,36 @@ bool pc_equipitem(struct map_session_data *sd,short n,int req_pos,bool equipswit
 	return true;
 }
 
+void pc_deleteautobonus( struct map_session_data& sd, std::vector<s_autobonus>& bonus, int position ){
```

Comment:
`struct map_session_data& sd` could be removed since sd is not used in the function?